src/charm.py

```diff
@@ -430,7 +443,7 @@ def _config_changed(self, event: HookEvent) -> None:
             container.add_layer(SERVICE_NAME, layer_config, combine=True)
             container.pebble.replan_services()
             self.ingress.update_config(self._make_ingress_config())
-            self._config_force_https()
+            # self._config_force_https()
```

Comment:
Can this line be removed rather than commented out? Generally it would be better not to have commented out code as it makes maintenance more difficult

Reply:
I was just testing some scenarios